android/src/main/java/com/instabug/reactlibrary/RNInstabugReactnativeModule.java

```diff
@@ -1082,6 +1089,105 @@ public void run() {
             }
         });
     }
+    /**
+     * Register a listener for W3C flags value change
+     */
+    @ReactMethod
+    public void registerW3CFlagsChangeListener(final Callback handler){
```

Comment:
We are not using this `handler` callback anywhere in the code so we don't need to send it to the native side, I think we can safely remove it.